### ISSUE TITLE
Build On New Architecture Stuck For Android & Never Succeed

### ISSUE THREAD

**Issue by ErHarinderSingh** [facebook/react-native#33657]
### Description

![Screenshot 2022-04-16 at 13 50 51](https://user-images.githubusercontent.com/22139572/163668042-b951cd7f-7710-4b9a-b259-c8f88bc4c490.png)

### Version

 Tried On 0.68.0 & 0.68.1

### Output of `npx react-native info`

![Screenshot 2022-04-16 at 13 57 50](https://user-images.githubusercontent.com/22139572/163668123-6959090f-2b06-4c40-a6f9-c137de62c064.png)

### Steps to reproduce
**I have MacBook Air 2017**
npx react-native init demo
cd demo
newArchEnabled=true in **gradle.properties** file
enable hermes
npx react-native run-android

**Comment by ErHarinderSingh** [comment]
I can verify that this issue is with **MacBook Air 2017**. I checked on **MacBook Pro M1 2020** and it's working fine with new architecture.
It means it is device specific.

**Comment by ErHarinderSingh** [comment]
I have wait and after 30 minutes debug build compiled now successfully. Now trying release build

**Comment by ErHarinderSingh** [comment]
Release build almost takes 40 minutes to build & install.
Unfortunately app is crashed after launching.

**Comment by ErHarinderSingh** [comment]
Crash Log shows this:
`D/SoLoader: libhermes.so not found on /data/data/`

**Comment by joaotmachado** [comment]
Our company is working for 3 weeks trying to enable the new architecture following all the instructions provide by the community. We tried on several Macs and Pcs. Unfortunately we are in eternal looping with those errors.

**Comment by ErHarinderSingh** [comment]
@joaotmachado I solved everything only start up app crash issue yet to resolved. It happens in both enabled/disabled hermes release builds. Debug builds are fine. First time build are taking too much time. 42 minutes on MacBook Air 2017. Is this new architecture? With lots of errors? Lots of conflicts? Lots of dependency errors? 🤦🏻‍♂️😞

**Comment by raajnadar** [comment]
> @joaotmachado I solved everything; only start up app crash issue yet to resolved. It happens in both enabled/disabled hermes release builds. Debug builds are fine. First time build are taking too much time. 42 minutes on MacBook Air 2017. Is this new architecture? With lots of errors? Lots of conflicts? Lots of dependency errors? 🤦🏻‍♂️😞

Are you trying on fresh project or existing upgraded project?

**Comment by cortinico** [comment]
Just a heads up here: the new architecture is building Native Code (C++ code) which is causing your build time to increase.

This is (sadly) expected. If you build on a MacBook Air, you will most likely experience longer build time.

Generally the first build is going to take longer, but subsequent builds will be faster as there is no need to rebuild everything.

We also wrote a page to help you speed up your build speed here:
https://reactnative.dev/docs/build-speed

> Is this new architecture? With lots of errors? Lots of conflicts? Lots of dependency errors? 🤦🏻‍♂️😞

As we wrote extensively on the website, the new architecture is still experimental. We're collecting feedback, like yours, to understand where to prioritize our efforts as we refine the new architecture for everyone.

Closing as this is essentially resolved.
